repos/system_upgrade/el7toel8/actors/authselectscanner/libraries/library.py

```diff
@@ -0,0 +1,105 @@
+import errno
+import re
+
+from leapp.libraries.stdlib import call
+from subprocess import CalledProcessError
+
+
+def read_file(file):
```

Comment:
If the purpose is to read a single file and return '' if there is None - maybe remove the try/catch and check file presence with other methods?

Reply:
Yes, this is the purpose. What would be a better way to do it?

Reply:
I'd substitute L16-L23 with smth like
```
if not os.path.exists(config) or not os.path.isfile(config):
    return ""
with open(config) as f:
    return f.read()
```
Please don't shadow file builtin, use another name like config/filepath/etc.
Also imho there is no point in raising an exception at L23.